src/python/pants/backend/pants_info/list_target_types.py

```diff
@@ -162,9 +173,14 @@ def format_for_cli(self, console: Console) -> str:
         type_info = console.cyan(f"{indent}type: {self.type_hint}, {required_or_default}")
         lines = [field_alias, type_info]
         if self.description:
-            lines.extend(f"{indent}{line}" for line in textwrap.wrap(self.description, 80))
+            lines.extend(f"{indent}{line}" for line in textwrap.wrap(self.description or "", 80))
         return "\n".join(f"{indent}{line}" for line in lines)
 
+    def as_dict(self) -> Dict[str, Any]:
+        d = dataclasses.asdict(self)
+        d.pop("alias")
```

Comment:
`del d['alias']` is better here since you don't need the return value from dict.pop